contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
+        altText="thumbnail image"
+        :appearanceOverrides="{
+          objectFit: thumbnailScaleType,
```

Comment:
[Required] Please remove this line and use `KImg`'s `scaleType` prop instead https://release-v3--kolibri-design-system.netlify.app/kimg/#prop:scaleType